force-app/main/default/classes/FlowControlLoopsCollections.cls

```diff
@@ -237,12 +300,9 @@ public with sharing class FlowControlLoopsCollections {
      * @return A Set of unique strings.
      */
     public static Set<String> createStringSet(List<String> inputList) {
-        // Create a new Set of strings
-        Set<String> stringSet = new Set<String>();
-
-        //add strings to the set
-
-        return null; // Replace null with the variable you used to store the result
+        // Create the Set with a List (i.e., pass the List into the Set constructor).  Since Sets can only hold unique values, this deduplicates the List    
+        Set<String> stringSet = new Set<String>(inputList);
+        return stringSet; // Replace null with the variable you used to store the result
```

Comment:
It isn't better or anything but for extra practice, try using the addAll() method to add the list to the set.